ql/src/java/org/apache/hadoop/hive/ql/reexec/ReExecutionDagSubmitPlugin.java

```diff
@@ -76,7 +76,7 @@ public void beforeExecute(int executionIndex, boolean explainReOptimization) {
   }
 
   @Override
-  public boolean shouldReExecute(int executionNum, CommandProcessorException ex) {
+  public boolean shouldReExecute(int executionNum) {
     return (executionNum < maxExecutions) && retryPossible;
```

Comment:
please change this to `retryPossible` ( and remove maxExecutions field)

Reply:
changed